js/w3c/style.js

```diff
@@ -31,6 +31,15 @@ define(
                     // with a PNG of the same base name. CONNEG must die.
                     css += "www.w3.org/community/src/css/spec/" + statStyle.toLowerCase() + ".css";
                 }
+                else if (statStyle === "webspec") {
+                    css = "https://specs.webplatform.org/assets/css/kraken.css";
+                    $('<link rel="shortcut icon" href="https://specs.webplatform.org/assets/img/icon.png">', doc)
```

Comment:
True, it's still in my coding reflexes from 15 years ago. Need to get those checked.